modules/local/sentieon/bwamem.nf

```diff
@@ -0,0 +1,67 @@
+process SENTIEON_BWAMEM {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    input:
+    tuple val(meta), path(reads)
+    path fasta
+    path fai
+    path index
+
+    output:
+    tuple val(meta), path('*.bam'), emit: bam
+    tuple val(meta), path('*.bai'), emit: bai
+    path  "versions.yml"          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def sentieon_exe = params.sentieon_install_dir ? "${params.sentieon_install_dir}/sentieon" : 'sentieon'
+    """
+    source sentieon_init.sh SENTIEON_LICENSE_BASE64
```

Comment:
I think this should be done at the config level with a beforescript directive: https://www.nextflow.io/docs/latest/process.html#beforescript

Reply:
Actually, I tried to do this one, but found this tricky. bin/ directory is in PATH when run as a part of the script section so its contents are automatically made accessible. But if I want to make it part of a beforeScript directive and move it to a config, I would need to give the path to the init script, and I am don't know how to do that. Is there an enviroment variable that I can use to get the path to the bin folder from the runtime? 🤔